source/transfer_list.rst

```diff
@@ -193,8 +192,12 @@ they're always expected to appear together on the first platform adding the tag
 (to encourage reusability in different situations). Alternatively, complex data
 may be represented in a different kind of well-established handoff data
 structure (e.g. FDT [DT]_, HOB [PI]_) that is inserted into the TL as a single
-TE. The same tag ID may occur multiple times in the TL to represent multiple
-instances of the same kind of object. Tag layouts (including the meaning of all
+TE. It is strongly recommended that entries of a given type are unique in the
```

Comment:
This strong recommendation should only be the case if the TE definition does not meet the requirements you list below. Just move to the end of the paragraph?